__tests__/mocks/api.js

```diff
@@ -20,7 +20,6 @@ export default {
     })
 
     return {
-      abort: () => {},
       promise
     }
```

Comment:
why do we need to return a hash then? this API was because we needed to return 2 things.

Reply:
Didn't want to change too many things. I can though.